x/authz/handle_msg.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 // HandleMsg implements modules.MessageModule
-func (m *Module) HandleMsg(index int, msg sdk.Msg, tx *juno.Tx) error {
+func (m *Module) HandleMsg(_ int, msg sdk.Msg, tx *juno.Tx) error {
```

Comment:
It can be also replaced by events.
See:
https://github.com/cosmos/cosmos-sdk/blob/release/v0.47.x/x/authz/keeper/keeper.go#L197
https://github.com/cosmos/cosmos-sdk/blob/release/v0.47.x/x/authz/keeper/keeper.go#L223